taqueria-plugin-tezos-client/client.ts

```diff
@@ -1,17 +1,53 @@
-import { sendAsyncErr } from '@taqueria/node-sdk';
-import { IntersectionOpts as Opts } from './common';
-import simulate from './simulate';
-import typecheck from './typecheck';
+import {
+	CmdArgEnv,
+	getArch,
+	getFlextesaImage,
+	sendAsyncErr,
+	sendErr,
+	sendRes,
+	sendWarn,
+	spawnCmd,
+} from '@taqueria/node-sdk';
+import { ClientOpts as Opts } from './common';
+
+const getArbitraryClientCmd = async (
+	parsedArgs: Opts,
+	userArgs: string,
+): Promise<CmdArgEnv> => {
+	const projectDir = process.env.PROJECT_DIR ?? parsedArgs.projectDir;
+	if (!projectDir) throw `No project directory provided`;
+	const arch = await getArch();
+	const flextesaImage = await getFlextesaImage(arch);
+	const binary = 'docker';
+	const baseArgs = [
+		'run',
+		'--rm',
+		'-v',
+		`${projectDir}:/project`,
+		'-w',
+		'/project',
+		'--platform',
+		arch,
+		flextesaImage,
+		'octez-client',
+	];
+	const processedUserArgs = userArgs.split(' ').map(arg => arg.startsWith('\\-') ? arg.substring(1) : arg).filter(arg =>
+		arg
+	);
+	const args = baseArgs.concat(processedUserArgs);
+	const envVars = {};
+	return [binary, args, envVars];
+};
+
+const runArbitraryClientCmd = (parsedArgs: Opts, cmd: string): Promise<string> =>
+	getArbitraryClientCmd(parsedArgs, cmd)
+		.then(spawnCmd)
+		.then(() => `Command "${cmd}" ran successfully by octez-client`)
```

Comment:
Same with this one. I'd omit the success msg, or if that's required, then to inspect the exit code and ensure its 0 before giving a success msg.